app/src/main/java/com/kickstarter/libs/utils/ApplicationLifecycleUtil.java

```diff
@@ -53,7 +53,7 @@ public void onActivityResumed(final @NonNull Activity activity) {
       this.client.config()
         .compose(Transformers.pipeApiErrorsTo(this::handleConfigApiError))
         .compose(Transformers.neverError())
-        .subscribe(this.config::config);
+        .subscribe(c -> this.config.config(c));
```

Comment:
it's an edge check because there's a `this.config` and a `config()` refresh method that it gets confused by :/